src/common/config_opts.h

```diff
@@ -13,7 +13,7 @@
  */
 
 /* note: no header guard */
-OPTION(host, OPT_STR, "localhost")
+OPTION(host, OPT_STR, "default")  // "default" means that host will be `hostname -s` 
```

Comment:
I think if we just change teh default from "localhost" to "", and change "" to fill in the real hostname, we'll be pretty safe:

- if someone configured a host, that won't change
- if someone is using $host but is getting the default of 'localhost', they probably haven't noticed.. and hopefully won't notice if it starts getting a real hostname.  We can add something to the release notes.
- Same for if 'host = ' and they are using $host